src/main/java/kayu/commands/EmptyCommand.java

```diff
@@ -0,0 +1,28 @@
+package kayu.commands;
+
+import static kayu.commands.CommandMessage.ERROR_EMPTY_COMMAND;
+import static kayu.commands.CommandType.EMPTY;
+
+import kayu.exception.DukeException;
+import kayu.service.TaskList;
+
+/**
+ * Represents a {@link kayu.commands.Command} that indicates that the user has inputted an empty command.
+ */
+public class EmptyCommand extends Command {
+
+    /**
+     * Initializes a EmptyCommand- {@link kayu.commands.Command}.
```

Comment:
Not a major issue, just a small grammatical error to use "an" instead of "a" before **EmptyCommand** in the Javadoc. Might want to ensure that the Javadoc is free of grammatical error. However, most of your Javadoc is already really well done, just a minor one here. 😄

Reply:
Resolved grammatical errors.